SignalServiceKit/protobuf/OWSSignalServiceProtos.proto

```diff
@@ -31,10 +31,22 @@ message Envelope {
 }
 
 message Content {
-  optional DataMessage dataMessage = 1;
-  optional SyncMessage syncMessage = 2;
-  optional CallMessage callMessage = 3;
-  optional NullMessage nullMessage = 4;
+  optional DataMessage    dataMessage = 1;
+  optional SyncMessage    syncMessage = 2;
+  optional CallMessage    callMessage = 3;
+  optional NullMessage    nullMessage = 4;
+  optional ReceiptMessage receiptMessage = 5;
+}
+
+message ReceiptMessage {
+  enum Type {
+    DELIVERY = 0;
+    READ     = 1;
+  }
+
+  optional Type   type       = 1;
+  repeated uint64 timestamp  = 2;
+  optional bytes  profileKey = 3;
```

Comment:
We're gonna be sending a lot of these - do we really want to include the profile key every time? It would seemingly dwarf the size of the rest of the message.

Reply:
Good point - and Moxie already asked me to remove this I just forgot to push the commit. Sorry!